### ISSUE TITLE
NodeGraphQt.Port.add_accept_port_type() 

### ISSUE THREAD

**Issue by snorthman** [jchanvfx/NodeGraphQt#342]
`Once a constrain has been added only ports of that type specified will be allowed a pipe connection.`

from the docs of the titular function.

I have (these are snips):

```python
class SimulationNode(BaseSimulatorNode):
   def __init__(self):   
      ...
      self.out_simulation = self.add_output('sim', color=PortType.SIMULATION.color())
      ...
```

and

```python
class SimulatorNode(BaseSimulatorNode):
   def __init__(self):   
        self.in_simulation = self.add_input('sim', display_name=False, color=PortType.SIMULATION.color())
        self.in_simulation.add_accept_port_type(port_name='sim',
                                                port_type=PortTypeEnum.OUT.value,
                                                node_type='simulator.SimulationNode')
```

My assumption is now, that my `SimulationNode.out_simulation` is the only port type that can connect to the `SimulatorNode.in_simulation`.

This is however, not the case, and everything is still accepted by `SimulatorNode.in_simulation`.

`NodeGraphQt.Port.add_reject_port_type()` does work in line with expectations, but its unfeasible to start rejecting every possible port except a particular port.

Is this a bug, or did I do something wrong?

**Comment by jchanvfx** [comment]
Hi @snorthman, 

Nope you have indeed found a bug :sweat_smile: and I will look into it 😃 

Cheers,
Johnny

**Comment by jchanvfx** [comment]
I've released v0.6.2 with the fix 😄 
https://github.com/jchanvfx/NodeGraphQt/releases/tag/v0.6.2
